incubator/hnc/internal/kubectl/tree.go

```diff
@@ -81,6 +81,21 @@ func printSubtree(prefix string, hier *api.HierarchyConfiguration, inCycle bool)
 		if cycle && inCycle {
 			continue
 		}
+
+		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+		defer cancel()
+
+		ns, err := k8sClient.CoreV1().Namespaces().Get(ctx, txt, metav1.GetOptions{})
+		if err != nil {
+			fmt.Printf("Could not list namespaces: %s\n", err)
+			os.Exit(1)
+		}
+
+		_, hasSubnamespaceAnnotation := ns.ObjectMeta.Annotations["hnc.x-k8s.io/subnamespaceOf"]
```

Comment:
You should almost never have raw strings in your programs. Use `api.SubnamespaceOf`, defined here: https://github.com/kubernetes-sigs/multi-tenancy/blob/master/incubator/hnc/api/v1alpha2/subnamespace_anchor.go#L27

Reply:
done